### PR TITLE
REP-4839 Removing Vagrant release verification/sandbox

### PR DESCRIPTION
**Pull request by dmnjohns**

### CODE REVIEW

File: repose-aggregator/tests/release-verification/README.md [hunk 1]
```diff
@@ -35,14 +35,14 @@ build the project locally and set up the built artifacts. A value of
 # Verifying A Release
 Gradle tasks have been set up to allow for verifying either the DEB
 packages, the RPM packages, or both. To verify a release, simply
-run `gradle smokeTest -Prelease-version=<version>` (e.g. 8.1.0.0) from
+run `gradle dockerSmokeTest -Prelease-version=<version>` (e.g. 8.1.0.0) from
```
Comment (wdschei) [inline repose-aggregator/tests/release-verification/README.md]: Why isn't this just staying `smokeTest`?
Update line 248?

File: repose-aggregator/tests/release-verification/README.md [hunk 2]
```diff
@@ -35,14 +35,14 @@ build the project locally and set up the built artifacts. A value of
...
 this project. If the build succeeds, then the release succeeded! Different
 versions require different configurations to test all of the artifacts.
 To verify a v7.x release from the main project dir, run something like
-`gradle :repose-aggregator:tests:release-verification:vagrantSmokeTest -Prelease-version=7.3.7.1 -Pconfig-dir=/Fully/Qualified/Path/repose/repose-aggregator/tests/release-verification/src/config_7`
+`gradle :repose-aggregator:tests:release-verification:dockerSmokeTest -Prelease-version=7.3.7.1 -Pconfig-dir=/Fully/Qualified/Path/repose/repose-aggregator/tests/release-verification/src/config_7`
```
Comment (wdschei) [inline repose-aggregator/tests/release-verification/README.md]: Same here?

File: repose-aggregator/tests/release-verification/README.md (hunk 1 above)
Comment (dmnjohns) [inline repose-aggregator/tests/release-verification/README.md]: Missed it. Will update.

File: repose-aggregator/tests/release-verification/README.md (hunk 2 above)
Comment (dmnjohns) [inline repose-aggregator/tests/release-verification/README.md]: Missed it. Will update.

**Comment by wdschei** [comment]
That was a weird test failure.

**Comment by wdschei** [comment]
Retest this please.

**Comment by dmnjohns** [comment]
Retest this, please.

**Pull request merged**